pkg/api/converterfromapi.go

```diff
@@ -191,6 +208,12 @@ func convertAgentPoolProfileToVLabs(api *AgentPoolProfile, p *vlabs.AgentPoolPro
 	p.VnetSubnetID = api.VnetSubnetID
 	p.SetSubnet(api.Subnet)
 	p.FQDN = api.FQDN
+	p.Secrets = []vlabs.KeyVaultSecrets{}
```

Comment:
the diff is doing weird things. this is only changing in the vlabs. the 20160330 is trailing lines on the previous change